signing/secp256k1/impl/src/test/java/tech.pegasys.signers.secp256k1.azure/AzureKeyVaultSignerProviderTest.java

```diff
@@ -12,52 +12,38 @@
  */
 package tech.pegasys.signers.secp256k1.azure;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import tech.pegasys.signers.secp256k1.api.Signer;
-
-import java.math.BigInteger;
-import java.util.Arrays;
-
-import com.microsoft.azure.keyvault.KeyVaultClient;
-import com.microsoft.azure.keyvault.models.KeyBundle;
-import com.microsoft.azure.keyvault.webkey.JsonWebKey;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.junit.jupiter.MockitoExtension;
-import org.web3j.crypto.ECKeyPair;
-import org.web3j.utils.Numeric;
 
 @ExtendWith(MockitoExtension.class)
 public class AzureKeyVaultSignerProviderTest {
 
   @Test
   public void generatedSignerHasExpectedAddress() {
+    /*
```

Comment:
this test should be deleted since it was validating previous methods interactions.

Reply:
removed